.github/workflows/l10n-gettext-extract.yml

```diff
@@ -43,19 +43,20 @@ jobs:
           cd ../..
 
           # Get hash from PR commit.
-          NEW_HASHES=$(python ../fxa-l10n/scripts/pot_checksum.py ./packages/fxa-content-server/locale/templates/)
+          NEW_HASHES=$(python ../fxa-l10n/scripts/pot_checksum.py ./packages/fxa-content-server/locale/templates/LC_MESSAGES)
```

Comment:
This change is not necessary.

What the script does is searching for `.pot` files recursively. What this change makes, is returning `{FILE}.pot` instead of `LC_MESSAGES/{FILE}.pot`.

So, instead of 

```
LC_MESSAGES/client.pot:b62ce764345c83f9a8b37d682936e26f4a0e17911b84b5d7b8ad6a24910def49
LC_MESSAGES/server.pot:d4d1c51ddc78ef843d7605cdec4b1bf67a3b3098fe7f065e5e52045c29b8b4be
```

You would get

```
client.pot:b62ce764345c83f9a8b37d682936e26f4a0e17911b84b5d7b8ad6a24910def49
server.pot:d4d1c51ddc78ef843d7605cdec4b1bf67a3b3098fe7f065e5e52045c29b8b4be
```

Since you're comparing the output of the script before and after, the change is not really important. What counts is clearing the folder if npx doesn't overwrite the files.